src/messaging/serializers.py

```diff
@@ -16,14 +16,21 @@ class Meta:
 
 
 class ThreadSerializer(WritableNestedModelSerializer):
-	profile_1 = ProfileSerializer()
-	profile_2 = ProfileSerializer()
```

Comment:
can we just keep these two in there? What if we want to serialize an arbitrary Thread?

Alternatively, you could rename this serializer to something more specific for just other profile.